### PR TITLE
Grd 543 adjustments

### PR DESCRIPTION
**Pull request by gavin-peng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +23,8 @@ java -jar cromwell.jar run starFusion.wdl --inputs inputs.json
 Parameter|Value|Description
 ---|---|---
 `inputFqs`|Array[Pair[File,File]]|Array of fastq read pairs
+`reference`|String|Version of reference genome
+`outputFilePrefix`|String|Prefix of outptu file
```
Comment (pruzanov) [inline README.md]: needs to be **outputFileNamePrefix**

File: README.md [hunk 2]
```diff
@@ -2,6 +2,8 @@
 
 Workflow that takes a fastq pair or optionally a chimeric file from STAR and detects RNA-seq fusion events.
 
+## Overview
+
```
Comment (pruzanov) [inline README.md:5]: empty sections do not need to be here

File: README.md [hunk 3]
```diff
@@ -49,6 +50,7 @@ Output | Type | Description
 `fusionCodingEffects`|File|Annotated fusion output tsv
 
 
+./commands.txt found, printing out the content...
```
Comment (pruzanov) [inline README.md]: This line is here by mistake?

File: vidarrtest-regression.json.in [hunk 4]
```diff
@@ -68,6 +68,8 @@
                 }
             ],
             "starFusion.runStarFusion.genomeDir": null,
+            "starFusion.reference": "hg38",
+            "starFusion.outputFilePrefix": "STAR-Fusion_golden_data",
             "starFusion.runStarFusion.jobMemory": null,
             "starFusion.runStarFusion.modules": null,
             "starFusion.runStarFusion.starFusion": null,
```
Comment (pruzanov) [inline vidarrtest-regression.json.in]: **modules**, **starFusion** and **genomeDir** should not be in RT

**Review by pruzanov — CHANGES_REQUESTED** [review]

File: vidarrtest-regression.json.in (hunk 4 above)
Comment (gavin-peng) [reply vidarrtest-regression.json.in]: done

File: README.md (hunk 3 above)
Comment (gavin-peng) [reply README.md]: removed

File: README.md (hunk 2 above)
Comment (gavin-peng) [reply README.md:5]: removed

File: README.md (hunk 1 above)
Comment (gavin-peng) [reply README.md]: fixed

**Review by pruzanov — APPROVED** [review]

**Pull request merged**